lib/client.js

```diff
@@ -141,7 +141,7 @@ class Client {
         url: this.pathUrl(path),
         rejectUnauthorized: !this.options.ignoreCerts,
         headers: {
-          'User-Agent': (options && options['User-Agent']) || 'openwhisk-client-js',
+          'User-Agent': (options && options['User-Agent']) || process.env['USER_AGENT'] || 'openwhisk-client-js',
```

Comment:
Environment variable name should follow existing configuration pattern for the library and start with `__OW` prefix, e.g. `__OW_USER_AGENT`. 
https://github.com/apache/incubator-openwhisk-client-js#environment-variables